### PR TITLE
The fist of the plasma star

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds martial arts, ported from paradise and tg station, which replace your regular unarmed attacks such as grabs, disarms or punches. They also have special moves with combos and etc.

Spessmen can learn them via scrolls/books. Traitors can get some of those scrolls via uplink for now.

Todo:

- [x] put them somewhere, maybe traitor uplinks

- [x] maybe adds more styles asking the lore team

- [x] balance?

### CODE REVIEW

**Comment by BurgerLUA** [comment]
While a cool concept, this is going to be a mess to balance if you're genuinely considering making this antagonist only and not something expensive if added to uplinks.

**Comment by OneOneThreeEight** [comment]
could you add krav maga and stuff for security officers to learn

**Comment by BurgerLUA** [comment]
Optional Cool Thing That Would Be Hard To Code:
In the holodeck, performing wrestling moves without training should be possible.

Also, if I perform a suplex on someone in the holodeck, will it do holo damage or actual brute damage?

**Comment by Ultrazundy** [comment]
Maybe replace krav with something else. Having 20th century martial art over 400 years in the future is dumb. Call is Solcom (Solarian Combat) or something.

**Comment by OneOneThreeEight** [comment]
Semantics. Street fighting and boxing existed in the latter half of the 19th century and it's popular in the 21st century. Rehashing the name of it just because it's offputting for the current century is just semanticist.

**Comment by Ultrazundy** [comment]
No I'm saying have a new martial art that supercedes krav maga, not rename krav  maga. Having krav maga in it is like having Kung fu, karate or jiu jitsu. They're not going to be relevant 400 years in the future against aliens with alien biology. You have a whole blank canvas to paint anything you want and you default to dull modern day stuff. Lame.

**Comment by OneOneThreeEight** [comment]
I don't think it's worthwhile getting into this debate. The name's fine. I don't see why it wouldn't be relevant as it takes its origin from human history and not xeno history.

**Comment by skull132** [comment]
No changelog nor description of who gets access to them. :ree:

**Comment by kevinz000** [comment]
i'm going to finally turn traitor on as security because of this :D CARPCURITY GO

File: code/datums/uplink/highly visible and dangerous weapons.dm [hunk 1]
```diff
@@ -74,6 +74,16 @@
 	item_cost = 5
 	path = /obj/item/clothing/gloves/claws
 
+/datum/uplink_item/item/visible_weapons/wrestling
+	name = "Wrestling Manual"
+	item_cost = 6
+	path = /obj/item/wrestling_manual
+
+/datum/uplink_item/item/visible_weapons/solcom
+	name = "SolCom Manual"
```
Comment (LordFowl) [inline code/datums/uplink/highly visible and dangerous weapons.dm]: Pathetic.

File: code/modules/martial_arts/unathi.dm [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+	F.teach(H)
+	to_chat(H, "<span class='notice'>You have learned the martial art of Kis Khan.</span>")
+	qdel(src)
+
+/datum/martial_art/kis_khan/proc/kis_khan_help()
+	set name = "Recall Teachings"
+	set desc = "Remember the martial techniques of the Kis-khan."
+	set category = "Kis-khan"
+
+	to_chat(usr, "<b><i>You hisses deeply and remembers the traditions...</i></b>")
```
Comment (LordFowl) [inline code/modules/martial_arts/unathi.dm]: ``You hiss``
``and remember``

File: code/modules/martial_arts/vaurca.dm [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+	F.teach(H)
+	to_chat(H, "<span class='notice'>You have learned the martial art of Vk'utet.</span>")
+	qdel(src)
+
+/datum/martial_art/vkutet/proc/vkutet_help()
+	set name = "Recall Teachings"
+	set desc = "Remember the martial techniques of the Vk'utet."
+	set category = "Vk'utet"
+
+	to_chat(usr, "<b><i>You chitters deeply and remembers the indoctrination...</i></b>")
```
Comment (LordFowl) [inline code/modules/martial_arts/vaurca.dm]: Subject verb disagreement on two cases.

File: code/modules/martial_arts/wrestling.dm [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+/datum/martial_art/wrestling/proc/wrestling_help()
+	set name = "Recall Teachings"
+	set desc = "Remember how to wrestle."
+	set category = "Wrestling"
+
+	to_chat(usr, "<b><i>You flex your muscles and have a revelation...</i></b>")
+	to_chat(usr, "<span class='notice'>Clinch</span>: Grab. Passively gives you a chance to immediately aggressively grab someone. Not always successful.")
+	to_chat(usr, "<span class='notice'>Suplex</span>: Disarm someone you are grabbing. Suplexes your target to the floor. Greatly injures them and leaves both you and your target on the floor.")
+	to_chat(usr, "<span class='notice'>Advanced grab</span>: Grab. Passively causes stamina damage when grabbing someone.")
```
Comment (LordFowl) [inline code/modules/martial_arts/wrestling.dm]: Should be clarified as pain damage, as the two are different on our codebase.

File: code/modules/martial_arts/unathi.dm [hunk 5]
```diff
@@ -0,0 +1,117 @@
+#define TAIL_SWEEP "HHD"
+#define SWIFT_DISARM "DDG"
+#define HAMMERING_STRIKE "DHD"
+
+/datum/martial_art/kis_khan
+	name = "Kis-khan"
```
Comment (LordFowl) [inline code/modules/martial_arts/unathi.dm]: Is this a Chris-chan reference?

File: code/modules/martial_arts/tajara.dm [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+	F.teach(H)
+	to_chat(H, "<span class='notice'>You have learned the martial art of Baqhrar.</span>")
+	qdel(src)
+
+/datum/martial_art/baghrar/proc/baghrar_help()
+	set name = "Recall Teachings"
+	set desc = "Remember the martial techniques of the Baghrar."
+	set category = "Baghrar"
+
+	to_chat(usr, "<b><i>You twitches your ears and remembers the techniques...</i></b>")
```
Comment (LordFowl) [inline code/modules/martial_arts/tajara.dm]: Subject verb disagreement on two counts.

File: code/modules/martial_arts/unathi.dm (hunk 5 above)
Comment (Alberyk) [reply code/modules/martial_arts/unathi.dm]: Name was given by lore people.

Comment (LordFowl) [reply code/modules/martial_arts/unathi.dm]: Goddamn jackboot.

**Pull request merged**